src/modules/notifications/lib/index.js

```diff
@@ -66,19 +66,24 @@ async function prepareNotification (filter, taskConfig, params, context = {}) {
  * @param {Array} contactData - data from prepare step above
  * @param {String} ref - unique reference for this message batch
  */
-async function sendNotification (queueManager, taskConfig, issuer, contactData, ref) {
+async function sendNotification (queueManager, taskConfig, issuer, contactData, ref, uniqueJobId) {
   // Create event
   const e = eventFactory(issuer, taskConfig, contactData, ref)
   await evt.save(e)
 
   logger.info(`Sending notification reference ${e.referenceCode} ID ${e.eventId}`)
 
   // Schedule messages for sending
+  let rowCount = 0
```

Comment:
If you are interested, there is a way we can get an index and not have to rely on a `rowCount` variable 🤷. Easier to demo rather than submit a suggestion though 😁